### PR TITLE
Add logging options to connection options

### PR DESCRIPTION
**Pull request by jhendrixMSFT**
Pipe the azcore.LogOptions through client connections.

### CODE REVIEW

File: src/generator/connection.ts [hunk 1]
```diff
@@ -53,6 +53,8 @@ function generateContent(session: Session<CodeModel>, exportClient: boolean): st
   text += '\tRetry azcore.RetryOptions\n';
   text += '\t// Telemetry configures the built-in telemetry policy behavior.\n';
   text += '\tTelemetry azcore.TelemetryOptions\n';
+  text += '\t// Logging configures the built-in logging policy behavior.\n';
+  text += '\Logging azcore.LogOptions\n';
```
Comment (catalinaperalta) [inline src/generator/connection.ts:57]: I think this is missing a `\t`

File: src/generator/connection.ts [hunk 2]
```diff
@@ -67,6 +69,7 @@ function generateContent(session: Session<CodeModel>, exportClient: boolean): st
     text += '\t\tRegisterRPOptions: armcore.DefaultRegistrationOptions(),\n';
   }
   text += '\t\tTelemetry: azcore.DefaultTelemetryOptions(),\n';
+  text += '\t\Logging: azcore.DefaultLogOptions(),\n';
```
Comment (catalinaperalta) [inline src/generator/connection.ts:72]: Same thing here??

File: src/generator/connection.ts (hunk 1 above)
Comment (jhendrixMSFT) [reply src/generator/connection.ts]: Fixed.

File: src/generator/connection.ts (hunk 2 above)
Comment (jhendrixMSFT) [reply src/generator/connection.ts]: Fixed.

**Review by catalinaperalta — APPROVED** [review]
LGTM

**Pull request merged**